Containerfile

```
@@ -1,24 +1,29 @@
 # This file contains the configuration for containerized esi-leap
 
 # Use python as the parent image for esi-leap
-FROM python:3.11
+FROM docker.io/python:3.11
 
 LABEL description="ESI-LEAP -- ESI lease policy manager"
 
+# Indicate that service listens on this port
+EXPOSE 7777
+
 # The working directory for esi-leap inside the container
 WORKDIR /esi-leap
 
-# Copy the files of esi leap inside the container
-COPY run_services.sh run_services.sh
+# Copy the requirements file and install dependencies
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
```

Re: my earlier comment, that would look like this:

```suggestion
COPY requirements.txt requirements.txt
RUN pip install -r requirements.txt
COPY . ./
RUN pip install .
```